Packs/Censys/Integrations/CensysV2/CensysV2.py

```diff
@@ -169,28 +206,214 @@ def search_certs_command(client: Client, args: Dict[str, Any], query: str, limit
         'flatten': False
     }
 
-    res = client.censys_search_certs_request(data)
-    results = res.get('results', {})[:limit]
+    raw_response = client.censys_search_certs_request(data).get('result', {}).get('hits')
+    if not raw_response or not isinstance(raw_response, list):
+        error_msg = f"Unexpected response: 'hits' path not found in response.result. Response: {raw_response}"
+        raise ValueError(error_msg)
+    results = raw_response[:limit]
     for result in results:
         contents.append({
-            'SHA256': result.get('parsed').get('fingerprint_sha256'),
-            'Issuer dn': result.get('parsed').get('issuer_dn'),
-            'Subject dn': result.get('parsed').get('subject_dn'),
-            'Names': result.get('parsed').get('names'),
-            'Validity': result.get('parsed').get('validity'),
+            'Issuer DN': result.get('parsed', {}).get('issuer_dn'),
+            'Subject DN': result.get('parsed', {}).get('subject_dn'),
+            'Validity not before': result.get('parsed', {}).get('validity_period', {}).get('not_before'),
+            'Validity not after': result.get('parsed', {}).get('validity_period', {}).get('not_after'),
+            'SHA256': result.get('fingerprint_sha256'),
+            'Names': result.get('names'),
             'Issuer': result.get('parsed').get('issuer'),
         })
 
-    human_readable = tableToMarkdown(f'Search results for query "{query}"', contents)
+    human_readable = tableToMarkdown(f'Search results for query "{query}"', contents, removeNull=True)
     return CommandResults(
         readable_output=human_readable,
         outputs_prefix='Censys.Search',
         outputs_key_field='fingerprint_sha256',
         outputs=results,
-        raw_response=res
+        raw_response=raw_response
     )
 
 
+def ip_command(client: Client, args: dict, params: dict):
+    fields = [
+        "labels", "autonomous_system.asn", "autonomous_system.name",
+        "autonomous_system.bgp_prefix", "autonomous_system.country_code",
+        "autonomous_system.description", "location.country_code",
+        "location.timezone", "location.province", "location.postal_code",
+        "location.coordinates.latitude", "location.coordinates.longitude",
+        "location.city","location.continent", "location.country", "services.service_name",
+        "services.port", "services.transport_protocol", "services.extended_service_name",
+        "services.certificate", "last_updated_at", "dns.reverse_dns.names",
+        "operating_system.source", "operating_system.part", "operating_system.version"
```

Comment:
Have we ensured that the number of hard-coded fields/labels fits all possible use cases without raising an exception?